src/python/pants/backend/python/dependency_inference/module_mapper.py

```diff
@@ -163,11 +163,3 @@ async def map_module_to_address(
     if first_party_address:
         return PythonModuleOwner(first_party_address)
     return PythonModuleOwner(address=None)
-
-
-def rules():
-    return [
-        map_first_party_modules_to_addresses,
-        map_third_party_modules_to_addresses,
-        map_module_to_address,
-    ]
```

Comment:
I'm not sure how I feel about this change. I think it's been a good convention that every file has its own `rules()` function. It makes it easier for people to remember to register their rule when making changes because of proximity.

Reply:
Ok. How would you feel about re-exporting them from `dependency_inference` at least?

Reply:
Absolutely. All in favor of that :)